content/blog/2017/2017-01-12-declarative-pipeline-beta-2.adoc

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: "Declarative Pipeline Syntax Beta 2 release"
+tags:
+- plugins
+- pipeline
+author: abayer
+---
+
+This week, we released the second beta of the new
+link:https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Model+Definition+Plugin[Declarative Pipeline syntax],
+available in the Update Center now as version 0.8.1 of _Pipeline: Model Definition_.
+You can read more about Declarative Pipeline
+link:/blog/2016/12/19/declarative-pipeline-beta/[in the blog post introducing the first beta]
+from December, but we wanted to update you all on the syntax changes in the
+second beta. These syntax changes are the last compatibility-breaking changes to
+the syntax for the 1.0 release planned for February, so you can safely start
+using the 0.8.1 syntax now without needing to change it when 1.0 is released.
+
+== Syntax Changes
+
+=== Agent
+In order to support more detailed and clear configuration of agents, as well as
+making agent syntax more consistent with the rest of the Declarative Pipeline
+syntax, we've moved the agent configuration into blocks. The `agent any` and
+`agent none` configurations work the same as previously, but `label`, `docker`
+and `dockerfile` now look like the following:
+
+[pipeline]
+----
```

Comment:
This block should be annotated with `// Declarative //` at the top, see [this doc](https://github.com/jenkins-infra/jenkins.io/blob/master/STYLEGUIDE.adoc#pipeline-code)